x-pack/test/functional/services/transform/transform_table.ts

```diff
@@ -201,6 +230,33 @@ export function TransformTableProvider({ getService }: FtrProviderContext) {
       }
     }
 
+    public async assertTransformRowActionEnabled(
+      action: 'Delete' | 'Start' | 'Stop' | 'Clone' | 'Edit',
+      expectedValue: boolean
+    ) {
+      const selector = `transformAction${action}`;
+      await retry.tryForTime(60 * 1000, async () => {
+        await browser.pressKeys(browser.keys.ESCAPE);
+        await testSubjects.click('euiCollapsedItemActionsButton');
```

Comment:
We should make sure that the actions button is clicked for the correct row by passing the transform id and using it like this:
```
await testSubjects.click(`transformListRow row-$transformId} > euiCollapsedItemActionsButton`);
```
Might be even worth to introduce a `rowSelector` method similar to what we have in the AD `job_table` service.
I know that some of the existing methods are also not using this and assuming that there's only one row present in the table. But I think we should make this more stable.

Reply:
Updated here https://github.com/elastic/kibana/pull/92314/commits/c6e7f20fa39c45ab29954a1e62c61f88d160d589